### PR TITLE
Aesry Hunting Support! Final?

### PR DESCRIPTION
**Pull request by elBlanquito**
This should work!

### CODE REVIEW

**Comment by elBlanquito** [comment]
Hunting area data for most mobs on Aesry!

File: data/base-hunting.yaml [hunk 1]
```diff
@@ -330,6 +369,61 @@ hunting_zones:
   - 6850
   - 6843
   - 6849
+  # https://elanthipedia.play.net/nipoh_oshu 240-350 Aesry
+  # 3 different areas here, didn't realize initially needs to be broken up:
+  nipoh_oshu
```
Comment (jandersson) [inline data/base-hunting.yaml]: Missing `:` after the name

File: data/base-hunting.yaml [hunk 2]
```diff
@@ -390,6 +484,12 @@ hunting_zones:
   - 9677
   - 9678
   - 9679
+  # https://elanthipedia.play.net/ghoul_crow 130-175
+  ghoul_crow
```
Comment (jandersson) [inline data/base-hunting.yaml]: Missing a `:` after the name

Comment (elBlanquito) [inline data/base-hunting.yaml]: THANK YOU!

File: data/base-hunting.yaml [hunk 3]
```diff
@@ -330,6 +369,61 @@ hunting_zones:
...
+  - 12508
+  - 12509
+  - 12480
+  - 12481
+  - 12501
+  - 12507
+  - 12506
+  - 12500
+  # https://elanthipedia.play.net/nightreaver_unyn 180-225 Aesry
+nightreaver_unyn:
```
Comment (rpherbig) [inline data/base-hunting.yaml]: `nightreaver_unyn:` here needs to be indented

Comment (elBlanquito) [inline data/base-hunting.yaml]: one last try #1821

**Pull request closed without merging**